GVFS/GVFS.Common/Database/PlaceholdersTable.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+
+namespace GVFS.Common.Database
+{
+    /// <summary>
+    /// This class is for interacting with the placeholders table in the SQLite database
+    /// </summary>
+    public class PlaceholdersTable : IPlaceholderCollection
+    {
+        private IGVFSConnectionPool connectionPool;
+
+        public PlaceholdersTable(IGVFSConnectionPool connectionPool)
+        {
+            this.connectionPool = connectionPool;
+        }
+
+        public static void CreateTable(IDbCommand command)

```

Comment:
I saw that this TODO was marked as complete:

> Validate diagnose works as expected

How does diagnose work with this change?  How will we be able to see the placeholder list entries?

Reply:
All the files in the `.gvfs\databases` directory will be copied to the diagnose file.  This will include the VFSForGit.sqlite file.  I have been using [DB Browser for SQLite](https://sqlitebrowser.org/) to look at this file and see the data.  It seems easy enough to use and browse/query the data.